README.md

```diff
@@ -41,6 +41,27 @@ We do not recommend letting the LLMs insert data or modify columns by itself, as
 LLMs to only allow 'SELECT' statements in the `cratedb_sql` tool, you can jailbreak this against
 our recommendation.
 
+# Install
+```shell
+uv pip install --upgrade git+https://github.com/crate/cratedb-mcp
+```
+
+# Usage
+Start MCP server with `stdio` transport.
+```shell
+uv run cratedb-mcp stdio
+```
+Start MCP server with `sse` transport.
+```shell
+uv run cratedb-mcp sse
+```
```

Comment:
Install vs. usage can also be compressed into a single `uv` incantation, if you like it.
```shell
uvx --with=git+https://github.com/crate/cratedb-mcp cratedb-mcp sse
```